<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="creationStack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>PointBuy 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PointBuy 15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PointBuy 20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PointBuy 25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Roll the dice! (4d6)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>d10+7 sorted</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>301</width>
        <height>190</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="2">
        <widget class="QLabel" name="pointsLeftLabel_2">
         <property name="text">
          <string>Points left:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="dexBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="intPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="conBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="strBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="chaBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="wisPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="chaPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="wisBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="dexPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="intBox_2">
         <property name="minimum">
          <number>7</number>
         </property>
         <property name="maximum">
          <number>18</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="strPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="conPointsNext_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="strPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="dexPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="conPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="intPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="wisPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="chaPointsTotal_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="leftPointsTotal_2">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Strength</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Dexterity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Constitution</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Intelligence</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Wisdom</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Charisma</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="rollDiceButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Roll the dice!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="nextButton_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose a Race</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Half Orc</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Dwarf</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Halfling</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Gnome</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Elf</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Half-Elf</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="radioButton_7">
         <property name="text">
          <string>Human</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="nextButton_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="nextButton_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="nextButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="nextButton_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="raceGroup"/>
 </buttongroups>
</ui>
